src/sentry/utils/audit.py

```diff
@@ -35,3 +48,63 @@ def create_audit_entry(request, transaction_id=None, logger=None, **kwargs):
         logger.info(entry.get_event_display(), extra=extra)
 
     return entry
+
+
+def create_org_delete_log(entry):
+    """ Creates a log of deleted organization """
```

Comment:
Same note about the comment here. Comments are great, but I feel like when the comment is literally reiterating the function name, it's just superfluous. So either remove or make this more useful. (I'm learning towards just removing it since it's pretty self explanatory.